docs/src/main/sphinx/release/release-384.md

```diff
@@ -0,0 +1,39 @@
+# Release 384 (3 June 2022)
+
+## General
+
+* Fix queries failing due to worker data exchange optimization introduced in
```

Comment:
I would just phrase it as:

> Fix potential query failure due to a regression introduced in version 383.

The fact that it's due to an optimization in exchanges is irrelevant to users that just want to know if the issue they are encountering is fixed.

We may also want to add a note in the 383 release notes saying:

> This version has a regression that may cause queries to fail.